lib/bundler/audit/cli.rb

```diff
@@ -34,6 +34,7 @@ class CLI < ::Thor
       method_option :verbose, :type => :boolean, :aliases => '-v'
       method_option :ignore, :type => :array, :aliases => '-i'
       method_option :update, :type => :boolean, :aliases => '-u'
+      method_option :db_path, :type => :string, :aliases => '-p', :default => File.join(Dir.home, '.local', 'share', 'ruby-advisory-db')
```

Comment:
I didn't notice this earlier but it looks like you can use `Bundler::Audit::Database::USER_PATH`.